patches/chrome-app-main_dll_loader_win.cc.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/chrome/app/main_dll_loader_win.cc b/chrome/app/main_dll_loader_win.cc
+index dd026800f3dee99c3fe48bd7d8a0e51bc3bddd72..f4bb381d8ce1050536c5a0248a5c5b8ac51a1e09 100644
+--- a/chrome/app/main_dll_loader_win.cc
++++ b/chrome/app/main_dll_loader_win.cc
+@@ -45,6 +45,10 @@
+ #include "content/public/common/content_switches.h"
+ #include "sandbox/win/src/sandbox.h"
+ 
++#if defined(BRAVE_CHROMIUM_BUILD)
++#include "brave/version.h"
++#endif
++
+ namespace {
+ // The entry point signature of chrome.dll.
+ typedef int (*DLL_MAIN)(HINSTANCE, sandbox::SandboxInterfaceInfo*, int64_t);
+@@ -95,7 +99,11 @@ base::FilePath GetModulePath(base::StringPiece16 module_name) {
+   // Othwerwise, return the path to the module in a versioned sub-directory of
+   // the current executable's directory. This is the expected location of
+   // modules for proper installs.
++#if defined(BRAVE_CHROMIUM_BUILD)
++  return exe_dir.AppendASCII(BRAVE_BROWSER_VERSION_FOUR_COMPONENTS).Append(module_name);
```

Comment:
this doesn't seem right to me. Everywhere else `kChromeVersion` will still be the chrome version so it seems we need to override chrome_constants.cc to make this consistent everywhere

Reply:
yep, `kChromeVersion` is used for the install directory in a lot of places so overriding to make it the brave version looks like the way to go to me.

Reply:
Do you mean overriding `chrome_version.h`?
If so, it's the same as changing `chrome/VERSION` file because `chrome_version.h` is generated from `chrome/VERSION`. Then, it would introduce inconsistency from many places because many other places refer `chrome/VERSION` directly.

The intention of this PR is avoiding touching `chrome/VERSION` and only applying brave version to install process.

Reply:
no, it isn't the same. chrome/VERSION is used in other places and kChromeVersion is used specifically for the install directory

Reply:
in either case this change won't work correctly because it will never load anything from this directory because the rest of the code uses kChromeVersion to identify the version directory

Reply:
kChromeVersion is also used by the mac installer so it is probably incorrect there

Reply:
chrome_paths_mac GetVersionedDirectory also uses kChromeVersion. I just don't see how this can work correctly if we don't change kChromeVersion

Reply:
Hmm, need to check this overriding doesn't affect other modules/platform because some other places use `chrome:kChromeVersion`.

And, this PR works fine. I tested install/uninstall from server and local and used many times.

Reply:
I think this PR only affect install process and finding the directory that includes proper `chrome.dll` on Windows.
Other platforms will not be affected.

Reply:
I'm not saying mac is affected by this PR, I'm saying the fact that we're using chrome's version in kChromeVersion make macos incorrect. The problems are related